index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link href="https://fonts.googleapis.com/css?family=Lato&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="resources/css/style.css">
+    <title>HTML SIGN UP FORM</title>
+</head>
+
+<body>
+    <nav class="main">
+        <div class="inner-nav">
+            <p>Already have an Intuit account?
+                    <a href="#"> I forgot my User ID or Password </a></p>
+            <button>
+                <img src="./resources/img/lock-icon.png" alt=" lock icon"> Sign In</button>
+        </div>
+    </nav>
+
+    <header class="header">
+        <nav class="logo">
+            <img src="resources/img/download.png" alt="Intuit logo">
+        </nav>
+        <ul>
+            <li><img src="./resources/img/mint.png" alt="Mint logo"> Mint</li>
+            <li><img src="./resources/img/quickbook.png" alt="QuickBooks logo"> Quickbooks</li>
+            <li><img src="./resources/img/turbo.jpeg" alt="Turbotax logo"> Turbotax</li>
+        </ul>
+    </header>
+
+    <main>
+        <div class="form">
+            <h1>Create an Intuit account</h1>
+        <h2>One account for everything Intuit, including Mint.</h2>
+        <a class="learn-more" href="#">Learn more</a>
+        <form action="/" method="POST">
+            <label>
```

Comment:
Note for label tags. They take a `for` attribute that references and input's `id` attribute. Example 
`<label for="email"> email </label> <input type='email' id='email'>  `